### PR TITLE
feat(ui): New Org Rate Limits

### PR DESCRIPTION
**Pull request by billyvg**
Requires:
- [x] #7295 

Review this commit range: https://github.com/getsentry/sentry/pull/7299/commits/613ca6a269c4f54ffb737b564e48b58ea7168c7e

+ UI Tests
+ Removes the old rate limits UI
New (in new settings):
image

New (in current settings):
image

Old:
image

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by billyvg** [comment]
Waiting for https://github.com/getsentry/sentry/pull/7269 to get `<PanelFieldItem>` for Account Limit max rate.

File: src/sentry/static/sentry/app/views/settings/organization/rateLimit/rateLimitView.jsx [hunk 1]
```diff
@@ -1,227 +1,131 @@
...
+                  'The maximum percentage of your account limit an individual project can consume.'
+                )}
+                step={5}
+                min={50}
+                max={100}
+                formatLabel={value => {
+                  return value !== 100 ? (
+                    `${value}%`
+                  ) : (
+                    <span dangerouslySetInnerHTML={{__html: 'No Limit &mdash; 100%'}} />
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/settings/organization/rateLimit/rateLimitView.jsx]: Should `No Limit` be translated?

**Review by benvinegar — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationRateLimits/index.jsx [hunk 2]
```diff
@@ -18,13 +17,11 @@ const OrganizationRateLimits = createReactClass({
...
     return (
+      // TODO(billy): Move to routes
       <LazyLoad
         component={() =>
-          getSettingsComponent(
-            () =>
-              import(/*webpackChunkName: "rateLimitView"*/ '../settings/organization/rateLimit/rateLimitView'),
-            () => import(/*webpackChunkName: "rateLimitView.old"*/ './rateLimitView.old'),
-            this.props.routes
+          import(/*webpackChunkName: "rateLimitView"*/ '../settings/organization/rateLimit/rateLimitView').then(
```
[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationRateLimits/index.jsx]: We are releasing the new views to everyone. They fit in with the old design just fine. That way we don’t need to have a huge release with all the new views at once.

[the remark this answers is not in the document]
Comment (benvinegar) [reply src/sentry/static/sentry/app/views/organizationRateLimits/index.jsx]: We should probably communicate this so nobody is surprised.

**Pull request merged**